src/components/ErrorBoundary/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,50 @@
+import '@testing-library/jest-dom/vitest';
+import { render, screen } from '@testing-library/react';
+import {
+  beforeEach,
+  describe,
+  expect,
+  test,
+  vi,
+  type MockInstance,
+} from 'vitest';
+import ErrorBoundary from './ErrorBoundary';
+
+describe('Error Boundary', () => {
+  let consoleMock: MockInstance<(...data: unknown[]) => void>;
+  beforeEach(() => {
+    consoleMock = vi.spyOn(console, 'log').mockImplementation(() => undefined);
```

Comment:
```suggestion
    consoleMock = vi.spyOn(console, 'error').mockImplementation(() => undefined);
```
React Error Boundaries логируют ошибки через console.error, а не console.log. тест мог проходить случайно или вообще не работать как ожидалось.